lib/ruby/signature/environment_loader.rb

```diff
@@ -93,6 +93,7 @@ def load(env:)
 
         if stdlib_root
           signature_files.push(*each_signature(stdlib_root + "builtin"))
+          signature_files.push(*each_signature(stdlib_root + "base64"))
```

Comment:
`"pathname"` and `"set"` should also be added, but in a separate pull request.

Reply:
I think we need better abstraction for requiring standard libraries. Ok for now.